sdcard/os/archlinux.sh

```diff
@@ -21,6 +23,8 @@ initos(){
             generaldownload;;
         cubietruck|bananapro)
             allwinnerdownload;;
+        odroid-c2)
```

Comment:
Don't know for sure [see.](http://unix.stackexchange.com/questions/101561/what-are-the-differences-between-bsdtar-and-gnu-tar). I obviously tried standard `tar`, but it failed. So I switched to the archlinux instructions.